wizard.py

```
@@ -623,41 +732,44 @@ def main():
     if 'advanced' in selected_services:
-        console.print("\n🗂️ [bold cyan]Obsidian/Neo4j Integration[/bold cyan]")
+        console.print("\n🗄️ [bold cyan]Neo4j Configuration[/bold cyan]")
+        console.print("Neo4j is used for Knowledge Graph (entity/relationship extraction from conversations)")
+        console.print()
+
+        # Always prompt for Neo4j password
+        while True:
+            try:
+                neo4j_password = console.input("Neo4j password (min 8 chars) [default: neo4jpassword]: ").strip()
+                if not neo4j_password:
+                    neo4j_password = "neo4jpassword"
```

Ruff (0.14.14)

[error] Possible hardcoded password assigned to: "neo4j_password"

(S105)